### ISSUE TITLE
Add single line namespace support

### ISSUE THREAD

**Issue by mihaipopescu** [uncrustify/uncrustify#1235]
namespace dirchanges { class DirectoryWatcher; }

Gets changed into this:

    namespace dirchanges
    { class DirectoryWatcher; }

Expected versus output diff:
---
```diff
diff --git "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\10053-UNI-1340.cpp" "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\10053-UNI-1340.cpp"
index 9b72c4f..9d2c895 100644
--- "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\10053-UNI-1340.cpp"
+++ "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\10053-UNI-1340.cpp"
@@ -1,6 +1,5 @@
-namespace dudeNamespace { class ForwardFooClass; }
+namespace dudeNamespace
+{ class ForwardFooClass; }
 
 namespace dudeNamespace
-{
-    class ForwardFooClass;
-}
+{ class ForwardFooClass; }
```

Debug support files:
---
[10053-UNI-1340.cpp](https://gist.githubusercontent.com/mihaipopescu/93aa32fd646458ae1c5696f3311a43c1/raw/9b72c4fc1efac4e40187585aa72147f89431e00b/10053-UNI-1340.cpp)
[10053-UNI-1340.cpp.log](https://gist.githubusercontent.com/mihaipopescu/93aa32fd646458ae1c5696f3311a43c1/raw/769892e1d9434d93f8250c5eec7deb17daa5d321/10053-UNI-1340.cpp.log)
[10053-UNI-1340.cpp.unc](https://gist.githubusercontent.com/mihaipopescu/93aa32fd646458ae1c5696f3311a43c1/raw/5e44ee784c1919677db092e0a576f47f351451c7/10053-UNI-1340.cpp.unc)
[Uncrustify.Cpp.cfg](https://gist.githubusercontent.com/mihaipopescu/93aa32fd646458ae1c5696f3311a43c1/raw/23d62d9dba5caf352fdc1793792e98eed4b81929/Uncrustify.Cpp.cfg)
[UNI-1340.cpp](https://gist.githubusercontent.com/mihaipopescu/93aa32fd646458ae1c5696f3311a43c1/raw/d2057cc46f999c9bd2c7cac4852f2eb4e7857601/UNI-1340.cpp)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by guy-maurel** [comment]
I get, with` # Uncrustify_d-0.65-152-9dcb3dc `:
```
namespace dudeNamespace
{ class ForwardFooClass; }

namespace dudeNamespace
{ class ForwardFooClass; }

namespace dudeNamespace
{
    class ForwardFooClass;
}
```
Please comment.

**Comment by mihaipopescu** [comment]
As you can see from the debug file [UNI-1340.cpp](https://gist.githubusercontent.com/mihaipopescu/93aa32fd646458ae1c5696f3311a43c1/raw/d2057cc46f999c9bd2c7cac4852f2eb4e7857601/UNI-1340.cpp)

The input is this:

```cpp
namespace dudeNamespace { class ForwardFooClass; }
```

And we expect the oneliner to be kept and not broken in several lines.

**Comment by guy-maurel** [comment]
Please test the proposal #1329

**Comment by mihaipopescu** [comment]
That looks good ! Thanks 👍

**Comment by gmaurel** [comment]
It is not a bug but a new feature request!

**Comment by mihaipopescu** [comment]
Agree, thanks!

**Comment by gmaurel** [comment]
Fixed at 35f88c9cefe046a1d75642ea740e674f9f718963, Jan 17 17:48:43 2018
